searchlib/src/tests/transactionlog/translogclient_test.cpp

```diff
@@ -298,29 +297,34 @@ fillDomainTest(Session * s1, const std::string & name)
     Packet b(DEFAULT_PACKET_SIZE);
     b.add(e2);
     b.add(e3);
-    EXPECT_EXCEPTION(b.add(e1), std::runtime_error, "");
-    ASSERT_TRUE (s1->commit(vespalib::ConstBufferRef(a.getHandle().data(), a.getHandle().size())));
-    ASSERT_TRUE (s1->commit(vespalib::ConstBufferRef(b.getHandle().data(), b.getHandle().size())));
-    EXPECT_EXCEPTION(s1->commit(vespalib::ConstBufferRef(a.getHandle().data(), a.getHandle().size())),
-                     std::runtime_error,
-                     "commit failed with code -2. server says: Exception during commit on " + name + " : Incoming serial number(1) must be bigger than the last one (3).");
-    EXPECT_EQUAL(a.size(), 1u);
-    EXPECT_EQUAL(a.range().from(), 1u);
-    EXPECT_EQUAL(a.range().to(), 1u);
-    EXPECT_EQUAL(b.size(), 2u);
-    EXPECT_EQUAL(b.range().from(), 2u);
-    EXPECT_EQUAL(b.range().to(), 3u);
+    EXPECT_THROW(b.add(e1), std::runtime_error);
+    bool commit_res = s1->commit(vespalib::ConstBufferRef(a.getHandle().data(), a.getHandle().size()));
+    assert(commit_res);
+    commit_res = s1->commit(vespalib::ConstBufferRef(b.getHandle().data(), b.getHandle().size()));
+    assert(commit_res);
+    try {
```

Comment:
Consider using the `VESPA_EXPECT_EXCEPTION` macro, found in <vespa/vespalib/gtest/gtest.h> header file.

Reply:
Fixed.